.github/workflows/cxx.yml

```diff
@@ -0,0 +1,36 @@
+name: 🥥 Lint C++
+
+on:
+  workflow_dispatch: {}
+  push:
+    paths:
+    - "**.cpp"
+    - "**.cc"
```

Comment:
Suggest we either add to the 'run'-list below or remove '.cc'/'.hpp' here to ensure they react on the same extensions.

Reply:
Sure, happy to remove `.cc` and `.hpp` here. [Paths](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths) reference in case it is useful - I added the other c++ extensions on top of what we have in the current working repo copy because I've seen them used in a couple commits, but you're right in that `run-clang-format.py` would never format files with those extensions as configured, since iirc we have to pass specific folders to it... I might try just passing it the root locally on this branch and if that doesn't work, will remove `.cc` and `.hpp` from the `paths` list which triggers this Action.